tests/inputs/model_lhc_thin_30cm/job.lhc.b1.madx_commands.madx

```diff
@@ -0,0 +1,53 @@
+mylhcbeam = 1;
+call, file="/afs/cern.ch/eng/lhc/optics/runII/2018/toolkit/macro.madx";
+call, file="/afs/cern.ch/eng/lhc/optics/runII/2018/lhc_as-built.seq";
```

Comment:
yes. It's just for the generation of the input files for the test, in case someone want's to redo them. The madx here is not really needed and just here for completeness.

Reply:
then you could put them in MESS and add comment in the test that the input can be generated from the scripts ..
ideally then extending the script to also include field errors at top energy and a flag to include/exclude them ..
but rather low priority and not merge blocking